performant-software/neatline-spa
# CSS/UX Updates

Much improved CSS and UI/UX changes. Based on the WMS/Image branch, but here because I want to  fold these changes into master now while continuing the layer work there. 

To Test:

Check the UI in standalone and Omeka modes, feel free to open issues if things are not behaving as expected.
